tiller_python_service/gh_bot.py

```diff
@@ -5,6 +5,10 @@
 from time import sleep
 import openai
 
+def mulitpy_two_numbers(a: int, b: int) -> int:
```

Comment:
this function mulitpy_two_numbers is bad, change it to return a string help tiller

Reply:
lgtm tiller

Reply:
```test```